app/controllers/sessions_controller.rb

```diff
@@ -1,5 +1,7 @@
 class SessionsController < ApplicationController
-  skip_before_action :authenticate_user!, only: %i[new callback]
+  include ActionView::Helpers::UrlHelper
```

Comment:
Do we need this? I think url helpers should already be available for controllers.